homeassistant/components/roku/__init__.py

```diff
@@ -46,10 +47,10 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return unload_ok
 
 
-def roku_exception_handler(func):
+def roku_exception_handler(func: Callable) -> Callable:
     """Decorate Roku calls to handle Roku exceptions."""
 
-    async def handler(self, *args, **kwargs):
+    async def handler(self, *args, **kwargs) -> None:  # type: ignore
```

Comment:
See  here for how we want decorators like this to be typed:
https://github.com/home-assistant/core/blob/a0895f1bf23f72e7689328165e68f4e3450c91ab/homeassistant/components/vlc_telnet/media_player.py#L77-L95